command/alias_test.go

```diff
@@ -252,3 +252,44 @@ func TestAliasSet_invalid_command(t *testing.T) {
 
 	eq(t, err.Error(), "could not create alias: pe checkout does not correspond to a gh command")
 }
+
+func TestAliasList_empty(t *testing.T) {
+	initBlankContext("", "OWNER/REPO", "trunk")
+
+	output, err := RunCommand("alias list")
+
+	if err != nil {
+		t.Fatalf("unexpected error: %s", err)
+	}
+
+	eq(t, output.String(), "")
+}
+
+func TestAliasList(t *testing.T) {
+	cfg := `---
+hosts:
+  github.com:
+    user: OWNER
+    oauth_token: token123
+aliases:
+  co: pr checkout
+  il: issue list --author=$1 --label=$2
+  clone: repo clone
+  prs: pr status
+  cs: config set editor 'quoted path'
+`
+	initBlankContext(cfg, "OWNER/REPO", "trunk")
+
+	output, err := RunCommand("alias list")
+
+	if err != nil {
+		t.Fatalf("unexpected error: %s", err)
+	}
+
+	test.ExpectLines(t, output.String(),
+		"clone:\trepo clone",
+		"co:\tpr checkout",
+		"il:\tissue list --author=\\$1 --label=\\$2",
+		"prs:\tpr status",
+		"cs:\tconfig set editor 'quoted path'")
```

Comment:
Since the output of this command should be easy to assert as a whole, I think we don't need `ExpectLines` here and can instead just compare `output.String()` to a static string. This has the added benefit of asserting that the lines appear in the right order as well.